src/plugins/intel_gpu/src/runtime/ocl/ocl_stream.cpp

```diff
@@ -309,25 +309,29 @@ event::ptr ocl_stream::enqueue_kernel(kernel& kernel,
     try {
         _command_queue.enqueueNDRangeKernel(kern, cl::NullRange, global, local, dep_events_ptr, set_output_event ? &ret_ev : nullptr);
     } catch (cl::Error const& err) {
-        /// WA: Force exit. Any opencl api call can be hang after CL_OUT_OF_RESOURCES.
-        if (err.err() == CL_OUT_OF_RESOURCES) {
-            ov::intel_gpu::ForceExit();
```

Comment:
this is intended behavior. Please do not change this behavior.

Reply:
Got it. Reverted.